lib/Echidna/Exec.hs

```diff
@@ -100,7 +100,7 @@ coveragePoints = sum . fmap S.size
 -- | Capture the current PC and codehash. This should identify instructions uniquely (maybe? EVM is weird).
 pointCoverage :: (MonadState x m, Has VM x) => Lens' x CoverageMap -> m ()
 pointCoverage l = use hasLens >>= \v ->
-  l %= M.insertWith (const . S.insert $ v ^. state . pc) (fromMaybe (W256 maxBound) $ h v) mempty where
+  l %= M.insertWith (const . S.insert $ (v ^. state . pc, True)) (fromMaybe (W256 maxBound) $ h v) mempty where
```

Comment:
are we sure we want to set `True` by default? see the `or` comment.

Reply:
We are using `Success` by default but it can be any value. Later, it will be updated to the result extracted from the Transaction execution.